cloud_governance/main/environment_variables.py

```diff
@@ -126,7 +127,14 @@ def __init__(self):
                                                                                                 '{}')
         self._environment_variables_dict['DAYS_TO_DELETE_RESOURCE'] = int(
             EnvironmentVariables.get_env('DAYS_TO_DELETE_RESOURCE', '7'))
-
+        cluster_prefix_default_value = ["kubernetes.io/cluster", "sigs.k8s.io/cluster-api-provider-aws/cluster"]
```

Comment:
Are u sure u want to hold it as a variable.
Why not have 1 input line:
self._environment_variables_dict['CLUSTER_PREFIX'] = "['kubernetes.io/cluster', 'sigs.k8s.io/cluster-api-provider-aws/cluster']"
and then parse it

Reply:
I used the try-except block

Reply:
try:
            self._environment_variables_dict['CLUSTER_PREFIX'] = json.loads(EnvironmentVariables.get_env(
                'CLUSTER_PREFIX',
                '["kubernetes.io/cluster", "sigs.k8s.io/cluster-api-provider-aws/cluster"]'
            ))
        except json.JSONDecodeError as err:
            self._environment_variables_dict['CLUSTER_PREFIX'] = [
                "kubernetes.io/cluster",
                "sigs.k8s.io/cluster-api-provider-aws/cluster"
            ]